worksheet_C/README.md

```diff
@@ -0,0 +1,4 @@
+
+1: If I am thinking correctly, the algorithm checks for duplicates in a list. If "i" and "j" are not at the same spot of the list and they are equal, then it counts as a duplicate.
+
+Completely clueless about the rest.
```

Comment:
Did Monday's session help to clarify? If not, please ask myself or your tutor for more detailed explanation.